qiime/metadata.py

```diff
@@ -14,12 +16,30 @@ def __init__(self, dataframe):
     def __repr__(self):
         return self._dataframe.to_json()
 
+    @staticmethod
+    def load(path):
+        try:
+            df = pd.read_csv(path, sep='\t', index_col=0, dtype=object)
+        except pd.io.common.CParserError:
+            raise ValueError(
+                'Metadata file format is invalid for file %s. Currently only '
+                'QIIME 1 sample metadata mapping files are officially '
+                'supported. These can be validated using [Keemei]'
```

Comment:
Markdown link in an error message is odd, how about `...validated using Keemei: http://keemei.qiime.org`?